src/pages/developers/learning-tools.tsx

```diff
@@ -0,0 +1,379 @@
+import PageMetadata from "@/components/PageMetadata"
+import Text from '@/components/OldText'
+import InfoBanner from "@/components/InfoBanner"
+import EnterpriseEth from '@/public/enterprise-eth.png'
+import LearningToolsCardGrid from "@/components/LearningToolsCardGrid"
+import { shuffle } from "lodash"
+import RemixImage from '@/public/dev-tools/remix.png'
+import EthDotBuildImage from '@/public/dev-tools/eth-dot-build.png'
+import ReplitImage from '@/public/dev-tools/replit.png'
+import ChainIDEImage from '@/public/dev-tools/chainIDE.png'
+import TenderlyImage from '@/public/dev-tools/tenderly.png'
+import CryptoZombieImage from "@/public/dev-tools/crypto-zombie.png"
+import EthernautImage from "@/public/dev-tools/oz.png"
+import CaptureTheEtherImage from "@/public/dev-tools/capturetheether.png"
+import ChainShotImage from "@/public/dev-tools/chainshot.png"
+import ConsensysImage from '@/public/dev-tools/consensys.png'
+import BloomTechImage from '@/public/dev-tools/bloomtech.png'
+import BuildSpaceImage from '@/public/dev-tools/buildspace.png'
+import QuestbookImage from '@/public/dev-tools/questbook.png'
+import MetaschoolImage from '@/public/dev-tools/metaschool.png'
+import NFTSchoolImage from '@/public/dev-tools/nftschool.png'
+import PointerImage from '@/public/dev-tools/pointer.png'
+import PlatziImage from '@/public/dev-tools/platzi.png'
+import SpeedRunEthereumImage from "@/public/dev-tools/speed-run-ethereum.png"
+import AlchemyUniversityImage from "@/public/dev-tools/alchemyuniversity.png"
+
+const Page = (props: ChildOnlyProp) => (
+  <Flex
+    direction="column"
+    align="center"
+    w="full"
+    mx="auto"
+    mt={16}
+    mb={0}
+    {...props}
+  />
+)
+
+const Header = (props: ChildOnlyProp) => (
+  <Flex
+    as="header"
+    direction="column"
+    align="center"
+    textAlign="center"
+    maxW="896px"
+    py={0}
+    px={8}
+    {...props}
+  />
+)
+
+const H1 = (props: ChildOnlyProp) => (
+  <Heading
+    as="h1"
+    color="text"
+    fontStyle="normal"
+    fontFamily="monospace"
+    textTransform="uppercase"
+    fontWeight="semibold"
+    fontSize="2rem"
+    lineHeight={1.4}
+    textAlign="center"
+    {...props}
+  />
+)
+
+const Subtitle = (props: HeadingProps) => (
+  <Heading
+    fontSize="xl"
+    lineHeight={1.4}
+    fontWeight="normal"
+    color="text300"
+    maxW="55ch"
+    mb={2}
+    mt={4}
+    {...props}
+  />
+)
+
+const SubtitleTwo = (props: ChildOnlyProp) => <Subtitle mt={0} {...props} />
+
+const ContentBox = (props: ChildOnlyProp) => (
+  <Box py={4} px={8} w="full" {...props} />
+)
+
+const StackContainer = (props: ChildOnlyProp) => (
+  <Box
+    border="1px"
+    borderColor="border"
+    borderRadius={{ base: 0, sm: "base" }}
+    w={{ base: "full", sm: "96%" }}
+    mx={{ base: 0, sm: 8 }}
+    my={8}
+    px={8}
+    py={12}
+    background="ednBackground"
+    {...props}
+  />
+)
+
+export const getStaticProps: GetStaticProps = async (
```

Comment:
Same comment as in the developers page PR, lets use the `satisfies GetStaticProps<Props>` form for consistency.